src/Parser/Hunter/BeastMastery/Modules/Spells/BestialWrath/BestialWrath.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatNumber } from 'common/format';
+import getDamageBonus from "Parser/Hunter/Shared/Core/getDamageBonus";
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import ITEMS from "common/ITEMS/HUNTER";
+
+const T204P_MODIFIER = 0.15;
+const T20_2P_MODIFIER_PR_STACK = 0.015;
+const BESTIAL_FURY_MODIFIER = 0.15;
+const BESTIAL_WRATH_BASE_MODIFIER = 0.25;
+let BESTIAL_WRATH_TOTAL_MODIFIER = 0;
+
+const AFFECTED_ABILITIES = [
+  SPELLS.KILL_COMMAND.id,
+  SPELLS.COBRA_SHOT.id,
+  SPELLS.MULTISHOT.id,
+];
+
+const debug = false;
+
+class BestialWrath extends Analyzer {
```

Comment:
This class is doing a whole lot, it's possible it should be split into several. What does the player gain from knowing the damage boost of using a baseline cooldown? I think you should split damage boost reports up by the different talents and tier bonuses that boost it, so the player knows what they're getting from each.

Reply:
I agree with the splitting it up, so I'll do that